### DISCUSSION TITLE
Disable modes

### DISCUSSION THREAD

**Discussion by NohWayJose** [gopro/labs#708] [Feature Requests]
See #709 

As a diver, there have been times where I've lost footage or a photo because I accidentally put the camera in time lapse, which I (personally) never need underwater. 

It would be great to have some checkboxes in the settings/QR app to exclude modes (or perhaps up-to two of the three modes)

**Comment by dnewman-gpsw** [comment]
I tried, but completely disabling modes is not something that can easily done with Labs.  I have tried a boot script that constantly set video mode automatically, but it affected some other camera operations.  In the next Labs release you can have script that detects what mode it is in, and perform an action, as a clearer solution.

```
<r0>v13mV!R    

//  if(not_recording)
//         if(mode > video_mode) 
//              set_video Mode(); 
//   goto repeat;

```
This is in the next release (couple of months away hopefully.)
